### PR TITLE
added support for user-defined XLA flags to test-maxtext.sh

### PR DESCRIPTION
**Pull request by kocchop**
1. added support for used defined XLA flags
2. changed source of PGO converter script to jax main

### CODE REVIEW

**Review by olupton — APPROVED** [review]

File: .github/container/test-maxtext.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ usage() {
     echo ""
     echo "  OPTIONS                    DESCRIPTION"
     echo "  -a, --additional-args      Additional fiddle args to pass to MaxText/train.py"
+    echo "  --model-name               Specify model to run. Example: llama2-7b, default"
```
Comment (olupton) [inline .github/container/test-maxtext.sh:16]: You also need to add this to the `getopt` call below; the CI jobs are failing with `getopt: unrecognized option '--model-name'`

**Review by olupton — CHANGES_REQUESTED** [review]

File: .github/container/test-maxtext.sh (hunk 1 above)
Comment (kocchop) [reply .github/container/test-maxtext.sh:17]: fixed it

File: .github/container/test-maxtext.sh [hunk 2]
```diff
@@ -13,10 +13,11 @@ usage() {
     echo ""
     echo "  OPTIONS                    DESCRIPTION"
     echo "  -a, --additional-args      Additional fiddle args to pass to MaxText/train.py"
+    echo "  --mem-fraction             Specify model to run. Example: 0.90, 0.85, 0.65"
```
Comment (olupton) [inline .github/container/test-maxtext.sh:16]: Description is wrong.

File: .github/container/test-maxtext.sh [hunk 3]
```diff
@@ -13,10 +13,11 @@ usage() {
...
+    echo "  --model-name               Specify model to run. Example: llama2-7b, default"
+    echo "  --attn-type                Specify model to run. Example: llama2-7b, default"
```
Comment (olupton) [inline .github/container/test-maxtext.sh:18]: Description and examples are wrong.

**Review by olupton — DISMISSED** [review]
A couple more comments on the documentation. Feel free to fix them later.

File: .github/container/test-maxtext.sh (hunk 3 above)
Comment (kocchop) [reply .github/container/test-maxtext.sh]: Thanks. corrected them.

**Pull request merged**